template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java

```diff
@@ -179,12 +179,17 @@ public static boolean isRazSupportedForCloudAndStack(CloudPlatform cloudPlatform
         return RAZ_ENABLED_CLOUD_PLATFORMS.getOrDefault(cloudPlatform, Collections.emptyList()).contains(stackType);
     }
 
-    public static boolean isRazTokenConfigurationSupported(String cdhVersion, CloudPlatform cloudPlatform, StackType stackType) {
-        LOGGER.info("ClouderaManagerRepo is compared for Raz Ranger token support in Data Hub and Data Lake");
+    public static boolean isRazConfigurationForServiceTypeSupported(String cdhVersion, CloudPlatform cloudPlatform, StackType stackType) {
+        LOGGER.info("Cloud Platform {} is checked for Raz Service type configuration in {} for cdhVersion {}", cloudPlatform, stackType, cdhVersion);
         return isVersionNewerOrEqualThanLimited(() -> cdhVersion, CLOUDERA_STACK_VERSION_7_2_10)
                 && isRazSupportedForCloudAndStack(cloudPlatform, stackType);
     }
 
+    public static boolean isRazConfigurationForRazRoleSupported(String cmVersion, CloudPlatform cloudPlatform, StackType stackType) {
```

Comment:
Updated the method name

Reply:
Maybe you did not get my point: Raz is supported by Azure and AWS. But this method is explicitly checking for GCP _only_

Reply:
yes. thats correct. this parameter only needs to be populated for GCP only. thats why I have renamed it to isRazConfigurationForRazRoleNeeded since it's only needed for GCP.